content/blog/2024/amerigeo-workshop/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Ephemeral Interactive Computing for AmeriGEO Workshop
+date: "2024-08-05"
+banner:
+  image: "featured.png"
```

Comment:
Not a big deal, but if you could grab a screenshot where my face isn't quite so contorted that would be great! If not no worries.

Reply:
I went back and https://github.com/2i2c-org/2i2c-org.github.io/pull/288/commits/4f495f10f2b9b9f274eb4879fd1465882220eb8f was the best I could do! Video screengrabs of people talking are hard 😅